index.qmd

```diff
@@ -0,0 +1,65 @@
+---
+title: "Arrow New Feature Showcase: `show_exec_plan()` "
```

Comment:
Why not call it by `explain()` or `show_query()`, since those are the names folks should be familiar with from dbplyr?